docs/operating-scylla/admin.rst

```diff
@@ -252,10 +252,9 @@ discard, or trim, any blocks no longer in use by the filesystem.
 Experimental Features
 =====================
 
-ScyllaDB Open Source uses experimental flags to expose non-production-ready features safely. These features are not stable enough to be used in production, and their API will likely change, breaking backward or forward compatibility.
+ScyllaDB uses experimental flags to expose non-production-ready features safely. These features are not stable enough to be used in production, and their API will likely change, breaking backward or forward compatibility.
 
 In recent ScyllaDB versions, these features are controlled by the ``experimental_features`` list in scylla.yaml, allowing one to choose which experimental to enable.
-For example, some of the experimental features in ScyllaDB Open Source 4.5 are: ``udf``, ``alternator-streams`` and ``raft``.
```

Comment:
These examples **except raft** are still true. I don't know if we want to list them, but they're true. Just the word "open source" and the mention of version 4.5 (!?) isn't  relevant.

Reply:
Ok. I do think we need a list of these experimental features and what they mean, and indeed this list needs to be kept up to date. I agree this probably needs to be a separate page (and it can be linked here), but it's not urgent.